api.js

```diff
@@ -40,6 +40,7 @@ class Api {
     this._strictMode = 'strictMode' in opts ? opts.strictMode : false
     this._magicCommandAdded = false
     this._modulesSeen = opts.modulesSeen || []
+    this._errorFormatter = undefined
```

Comment:
Maybe we could move this to the `configure` method, so there's an alternate way of setting/overriding it without calling `.unexpectedErrorFormatter(fn)`.

Perhaps something like this:
```js
this._errorFormatter = opts.errorFormatter || this._errorFormatter // expects a function
```